src/main/java/software/amazon/awssdk/crt/io/EventLoopGroup.java

```diff
@@ -58,6 +58,58 @@ private void onCleanupComplete() {
 
     public CompletableFuture<Void> getShutdownCompleteFuture() { return shutdownComplete; }
 
+
+    /*
+     * Static interface for access to a default, lazily-created event loop group for users who don't
+     * want to deal with the associated resource management.  Client bootstraps will use this event loop
+     * group if they are passed a null value.
+     */
+
+    /**
+     * Sets the number of threads for the static default event loop group, should it ever be created.  Has no
+     * effect if the static default event loop group has already been created.
+     *
+     * @param numThreads number of threads for the static default event loop group
+     */
+    public static void setStaticDefaultNumThreads(int numThreads) {
+        synchronized (EventLoopGroup.class) {
+            staticDefaultNumThreads = Math.max(1, numThreads);
```

Comment:
is this the right default? Should we default to num cores? using 0 would do the trick.

Reply:
Default is num cores; this is how to override default when you don't like it but still don't want to manage your own elg.